week5/Q1_test.py

```diff
@@ -0,0 +1,18 @@
+import Q1
+import unittest
+
+
+class AlphabetTest(unittest.TestCase):
+
+	def testBaseCase(self):
```

Comment:
Check a few more cases here.
Create some cases where words only different at a deeper level for example.
You can also create tests for very specific things like:
- two words that only differ at a very deep level (e.g., the first 10 characters are the same)
- two words that are one the prefix of the other
- three words that actually create an invalid ordering (because it requires A < B and B < A both)